diff -rupN linux-6.6.106/arch/arm64/boot/dts/rockchip/rk3328-nanopi-neo3.dts linux-6.6.106-patched/arch/arm64/boot/dts/rockchip/rk3328-nanopi-neo3.dts
--- linux-6.6.106/arch/arm64/boot/dts/rockchip/rk3328-nanopi-neo3.dts	2025-12-15 23:25:26.120670069 +0100
+++ linux-6.6.106-patched/arch/arm64/boot/dts/rockchip/rk3328-nanopi-neo3.dts	2025-12-15 23:41:52.904749138 +0100
@@ -8,6 +8,7 @@
 
 #include <dt-bindings/input/input.h>
 #include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
 #include "rk3328-dram-renegade-timing.dtsi"
 #include "rk3328.dtsi"
 
@@ -78,8 +79,17 @@
 		pinctrl-names = "default";
 
 		status_led: led-0 {
-			gpios = <&gpio0 RK_PA2 GPIO_ACTIVE_HIGH>;
 			label = "nanopi-neo3:green:status";
+			gpios = <&gpio0 RK_PA2 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "heartbeat";
+			color = <LED_COLOR_ID_RED>;
+		};
+
+		power_led: led-1 {
+			label = "nanopi-neo3:red:power";
+			gpios = <&gpio3 RK_PD2 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "none";
+			color = <LED_COLOR_ID_GREEN>;
 		};
 	};
 
@@ -341,6 +351,9 @@
 		status_led_pin: status-led-pin {
 			rockchip,pins = <0 RK_PA2 RK_FUNC_GPIO &pcfg_pull_none>;
 		};
+		power_led_pin: power-led-pin {
+			rockchip,pins = <0 RK_PD2 RK_FUNC_GPIO &pcfg_pull_none>;
+		};
 	};
 
 	pmic {
